gitlab/packages/gitlab/gitlab_stats.js

```diff
@@ -0,0 +1,84 @@
+// jshint esversion: 9
+
+/**
+ * @description null
+ * @param {ParamsType} params list of command parameters
+ * @param {?string} commandText text message
+ * @param {!object} [secrets = {}] list of secrets
+ * @return {Promise<SlackBodyType>} Response body
+ */
+
+const axios = require('axios');
+
+async function fetchStats(url) {
+    return await axios.get(url)
+    .then(response => { return response.data; })
+    .catch(error => { return error.response.data; });
+}
+
+async function _command(params, commandText, secrets = {}) {
+  const {
+    repo
+  } = params;
+  
+  // The gitLab api endpoint for viewing information on projects
+  const repoURL = `https://gitlab.com/api/v4/projects/${repo.replace(/\//g, "%2F")}`;
+  
+  // A dictionary of api urls for each statistic
+  const urlKey = {
+    fetches: `${repoURL}/statistics?access_token=${secrets.AcessToken_GitLab}`,
+    commits: `${repoURL}/repository/commits?access_token=${secrets.AcessToken_GitLab}`,
+    issuesOpened: `https://gitlab.com/api/v4/issues?state=opened&access_token=${secrets.AcessToken_GitLab}`,
+    issuesClosed: `https://gitlab.com/api/v4/issues?state=closed&access_token=${secrets.AcessToken_GitLab}`,
+    mergeRequestsOpened: `${repoURL}/merge_requests?state=opened&access_token=${secrets.AcessToken_GitLab}`,
+    mergeRequestsClosed: `${repoURL}/merge_requests?state=closed&access_token=${secrets.AcessToken_GitLab}`,
+    mergeRequestsLocked: `${repoURL}/merge_requests?state=locked&access_token=${secrets.AcessToken_GitLab}`,
+    mergeRequestsMerged: `${repoURL}/merge_requests?state=closed&access_token=${secrets.AcessToken_GitLab}`,
+  };
+  
+  // If the first GET request returns an error message stop everything and return it
+  var fetches = await fetchStats(urlKey.fetches);

```

Comment:
When the network request fails, we wouldn't be able to find the error. So keep it inside the `try` block.